src/main/java/uk/gov/pay/ledger/transaction/search/common/TransactionSearchParams.java

```diff
@@ -447,67 +449,65 @@ public String buildQueryParamString(Long forPage) {
         List<String> queries = new ArrayList<>();
 
         if (accountIds != null && !accountIds.isEmpty()) {
-            queries.add(GATEWAY_ACCOUNT_EXTERNAL_FIELD + "=" + String.join(",", accountIds));
+            queries.add(GATEWAY_ACCOUNT_EXTERNAL_FIELD + "=" + URLEncoder.encode(String.join(",", accountIds), StandardCharsets.UTF_8));
         }
         if (isNotBlank(fromDate)) {
-            queries.add(FROM_DATE_FIELD + "=" + fromDate);
+            queries.add(FROM_DATE_FIELD + "=" + URLEncoder.encode(fromDate, StandardCharsets.UTF_8));
         }
         if (isNotBlank(toDate)) {
-            queries.add(TO_DATE_FIELD + "=" + toDate);
+            queries.add(TO_DATE_FIELD + "=" + URLEncoder.encode(toDate, StandardCharsets.UTF_8));
         }
-
         if (isNotBlank(email)) {
-            queries.add(EMAIL_FIELD + "=" + email);
+            queries.add(EMAIL_FIELD + "=" + URLEncoder.encode(email, StandardCharsets.UTF_8));
```

Comment:
Can you do a manual test that searching for an email address works? I worry about the `@` sign!

Reply:
As above, will add this to the manual testing notes👌